lib/api/completeMultipartUpload.js

```diff
@@ -406,6 +407,11 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                 multipart: true,
                 log,
             };
+            const replicationInfo = getReplicationInfo(objectKey, destBucket,
```

Comment:
I will update this. I had considered that, but thought it would be better to not assign a property as `undefined`.